website/content/docs/extending-waypoint/plugin-interfaces/platform-releaser.mdx

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+page_title: 'PlatformReleaser'
+description: |-
+  How to implement the PlatformReleaser component for a Waypoint plugin
+---
+
+# PlatformReleaser
+
+https://pkg.go.dev/github.com/hashicorp/waypoint-plugin-sdk/component#PlatformReleaser
+
+The PlatformReleaser interface is an optional interface that a Platform component
+can implement to provide default Release functionality. This only takes effect
+if no release is configured. Therefore, users cannot set custom configurations
```

Comment:
```suggestion
if no release is configured, therefore users cannot set custom configurations
```

IMO these read like 3 half-sentances in a row, combining these two keeps the chain of thought going